opa/versions.py

```diff
@@ -1,12 +1,12 @@
 versions = [
     {
-        "product": "0.61.0",
+        "product": "0.66.0",
         "vector": "0.39.0",
```

Comment:
I think we need to update Vector first (if it gets updated)

Reply:
IMHO it's fine to first bump OPA and vector afterwards

Reply:
Oh yeah, it will just be a new nightly to replace the one created by this.

_I didn't see your review, and updated the description to add Vector as a dependency. Will leave it there until I come back to this (maybe tomorrow)._

Reply:
Honestly, I didn't fully get it. I leave it to you, I'm happy to either merge or wait :)

Reply:
I was just agreeing with you that it doesn't matter if they're done separately (this early), because:

1. Once this is merged, it will build the dev image with old Vector
2. Once Vector is updated, a new nightly will be built since the dev workflow would trigger it:
   https://github.com/stackabletech/docker-images/blob/056274f976de333caeb9ac9be4b3b8686109131f/.github/workflows/dev_opa.yaml#L18

The last line in _italics_ was just meaning I will wait because it was late and I didn't want to rush a merge. Now I'm happy to merge it and let it rebuild once Vector gets updated.